### PR TITLE
[CI] Move to GitHub actions

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by iHiD — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
CC @exercism/github-actions

**Review by SleeplessByte — APPROVED** [review]
❤️

**Review by cmccandless — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@d7b76f909a9ce4f0102c606459aa23439865bd38 # 1.50.0
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
```
Comment (SaschaMann) [inline .github/workflows/ci.yml:19]: Ruby is preinstalled: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#language-and-runtime

Do we need a specific version or could we just use the default version?

Comment (ErikSchierboom) [reply .github/workflows/ci.yml:19]: Unfortunately the default version doesn't work. See [this run](https://github.com/ErikSchierboom/website-copy/runs/1383907454?check_suite_focus=true) on my fork.

**Pull request merged**